contracts/peripheral/Strategies/UniSwap.sol

```diff
@@ -60,9 +60,8 @@ contract UniSwapStrategy is IStrategy {
             uint256 amountADesired,
             uint256 amountBDesired,
             uint256 amountAMin,
-            uint256 amountBMin,
-            uint256 slippage
```

Comment:
Everything would work well without slippage but not every time which is tied to reserves of tokens A and B. So let's keep that so that if such a case happens everything still works fine.